src/main/java/com/minecolonies/coremod/client/render/RenderFishHook.java

```diff
@@ -90,19 +90,19 @@ public void doRender(@NotNull final EntityFishHook entity, final double x, final
             @NotNull final Vec3d Vec3d = new Vec3d(-0.36D, 0.03D, 0.35D);
 
             Vec3d.rotatePitch((float) (-((double) citizen.prevRotationPitch + ((double) citizen.rotationPitch - (double) citizen.prevRotationPitch) * partialTicks)
-                                         * Math.PI / Literals.HALF_CIRCKLE));
+                                         * Math.PI / Literals.HALF_CIRCLE));
             Vec3d.rotateYaw((float) (-((double) citizen.prevRotationYaw + ((double) citizen.rotationYaw - (double) citizen.prevRotationYaw)
-                                                                            * partialTicks) * Math.PI / Literals.HALF_CIRCKLE));
+                                                                            * partialTicks) * Math.PI / Literals.HALF_CIRCLE));
             Vec3d.rotateYaw((float) (finalOrientation * 0.5D));
             Vec3d.rotatePitch((float) (-finalOrientation * 0.7D));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 0.7D to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)